ui/components/KustomizationDetail.tsx

```diff
@@ -26,12 +26,14 @@ const Info = styled.div`
 
 const TabContent = styled.div`
   margin-top: 52px;
+  width: 100%;
+  height: 100%;
 `;
 
 export default function KustomizationDetail({ kustomization, name }: Props) {
   const hashHistory = createHashHistory();
   return (
-    <>
+    <Flex wide column height="100%">
```

Comment:
Weird sort-of-inline styling here. I would expect `style={{ height: '100%' }}`.

Reply:
It's a prop - I didn't know that we had that but it's there. It was messing with my attempt at inline styling so if it's weird we should get rid of it

Reply:
Agreed